### ISSUE TITLE
Two shows refuse to load/fetch metadata even after doing a plex dance. Had metadata prior to losing it.

### ISSUE THREAD

**Issue by randomdice101** [ZeroQI/Hama.bundle#569]
I've had a weird bug occur that suddenly erased all my metadata, marked all shows as if being added the same recent day. The server slowly fetched and reloaded all metadata but for some reason Mahou Tsukai Precure (Anidb-11794) and Detective Conan (Anidb-266) refuse to load metadata. Prior to the bug the two shows loaded metadata fine. I've tried to remove the shows, clear cache, empty my trash and nothing seems to fix this issue.

![image](https://github.com/ZeroQI/Hama.bundle/assets/4016932/f75b8a84-2b48-4819-855c-614263940492)
[名探偵コナン [anidb-266].agent-search.log](https://github.com/user-attachments/files/15525314/anidb-266.agent-search.log)
[名探偵コナン [anidb-266].agent-update.log](https://github.com/user-attachments/files/15525315/anidb-266.agent-update.log)
[魔法つかいプリキュア! [anidb-11794].agent-search.log](https://github.com/user-attachments/files/15525316/anidb-11794.agent-search.log)
[魔法つかいプリキュア! [anidb-11794].agent-update.log](https://github.com/user-attachments/files/15525317/anidb-11794.agent-update.log)
I've added the logs I could find at %plex location%\Plex Media Server\Plug-in Support\Data\com.plexapp.agents.hama\DataItems\_Logs\Anime (25TB) 

Hopefully that is the right log? Please let me know if there is another log you require.

**Comment by ZeroQI** [comment]
both search function worked ok
"[名探偵コナン [anidb-266]]" log is truncated, as if it crashed but no crash error logged
"魔法つかいプリキュア! [anidb-11794]" is complete with all metadata, so you have a Plex issue, possibly corrupted database

https://support.plex.tv/articles/repair-a-corrupted-database/

**Comment by randomdice101** [comment]
Is there somewhere I could see the crash log for conan?

**Comment by ZeroQI** [comment]
It should be logged there, maybe restart plex refresh metadata for conan manually and it should be in hama agent log in l9gs folder. I did separate to create per seties log as hama logs were generally two concurrent updates logged simultaneously so was unreadable as a result

**Comment by randomdice101** [comment]
Okay let me give that a shot. then ill check the corrupted database steps

**Comment by randomdice101** [comment]
Produced the same log I'll try and fix the db hang on

**Comment by randomdice101** [comment]
Do you know a way to find the Plex SQLITE.exe? it seems my installation does not have it

**Comment by randomdice101** [comment]
I've found the tool, I'm trying to use it to fix the db but nothing seems to be wrong per the report

**Comment by randomdice101** [comment]
I've run all the steps, rebuilt the database, i'm still getting the same error. any way to fix this or logs you could help me check? I don't underestand why its refusing to load any metadata

**Comment by ZeroQI** [comment]
Try another agent to prove if it is plex or agent specific. It should show an error on the series specific logs or hama.log.... Do not believe it is related to the agent but could be wrong

**Comment by randomdice101** [comment]
Like add it in a folder that uses the plex agent then it should fail to find it? Or I can set the agent specific to the show? Just don’t want to change the whole library away

On Jun 2, 2024, at 14:24, Benjamin Brisson ***@***.***> wrote:

﻿

Try another agent to prove if it is plex or agent specific. It should show an error on the series specific logs or hama.log.... Do not believe it is related to the agent but could be wrong

**Comment by ZeroQI** [comment]
Create another library with another custom agent or thetvdb (not the new type libraries, they don't support third party agents, point to a folder with a series folder in it with videos inside. If it doesn't work it a plex issue. Change thz agent to hama, check if it updates metadata and if logs are full or show an error mesage...

**Comment by randomdice101** [comment]
I just made a new library with thetvdb and move it to its folder. without having to do anything metadata seems to have loaded.
![image](https://github.com/ZeroQI/Hama.bundle/assets/4016932/63791517-e416-4e2c-9b07-654084e7a6ba)

**Comment by randomdice101** [comment]
I also made a new test library using hamatv and it finds it okay???? 
![image](https://github.com/ZeroQI/Hama.bundle/assets/4016932/231e9844-3579-469c-a689-6a06161408c2)
![image](https://github.com/ZeroQI/Hama.bundle/assets/4016932/126db3ac-566c-47df-a5e8-1c1900a6aa96)
So how can i fix the other entry?

**Comment by ZeroQI** [comment]
Your library is corrupted, do the clean bundle and optimize actions, and
see...
It might have to be recreated with the same folder, but not an hama issue i
am afraid

Le dim. 2 juin 2024 à 22:52, randomdice101 ***@***.***> a
écrit :

**Comment by randomdice101** [comment]
I did the clean bundel and optimize actions as well prior to this. it didn't fix it either... if i recreate with the same folder I'll lose my watch history right?

**Comment by randomdice101** [comment]
Detective conan didn't load metadata however 
![image](https://github.com/ZeroQI/Hama.bundle/assets/4016932/fde5e802-b488-433d-8655-e20e8673f7aa)
Same issue as the other one

**Comment by ZeroQI** [comment]
I am not sure but believe so...
You need to fix the plex database corruption...

**Comment by ZeroQI** [comment]
Weird, but without logs with error message, difficult to create a fix and unlikely hama is at fault

**Comment by randomdice101** [comment]
Yeah I hit refresh metadata ill post the output once its done. same log as before just from the new library right?

**Comment by ZeroQI** [comment]
Hamlog and custom series log in case any of them have an error logged

**Comment by randomdice101** [comment]
Where is the custom series log? I know for the hame one

**Comment by ZeroQI** [comment]
Title.agent.log yiu gave it previously

**Comment by randomdice101** [comment]
[名探偵コナン [anidb-266].agent-search.log](https://github.com/user-attachments/files/15527528/anidb-266.agent-search.log)
[名探偵コナン [anidb-266].agent-update.log](https://github.com/user-attachments/files/15527529/anidb-266.agent-update.log)
[魔法つかいプリキュア! [anidb-11794].agent-search.log](https://github.com/user-attachments/files/15527530/anidb-11794.agent-search.log)
[魔法つかいプリキュア! [anidb-11794].agent-update.log](https://github.com/user-attachments/files/15527531/anidb-11794.agent-update.log)

Seems like its having the same issue as before. as for the Precure series though its smaller (158kb versus 208kb)

**Comment by ZeroQI** [comment]
11794 seem to work ok
266 crash while building AniDBTVDBMap but did a fix for that, please update hama and refresh metadata
